yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1014,6 +1015,7 @@ func fetchUnsupportedPGFeaturesFromSchemaReport(schemaAnalysisReport utils.Schem
 	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport(queryissue.SYSTEM_COLUMNS_NAME, queryissue.SYSTEM_COLUMNS_NAME, "", schemaAnalysisReport, false, ""))
 	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport(queryissue.LARGE_OBJECT_FUNCTIONS_NAME, queryissue.LARGE_OBJECT_FUNCTIONS_NAME, "", schemaAnalysisReport, false, ""))
 	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport(REGEX_FUNCTIONS_FEATURE, "", queryissue.REGEX_FUNCTIONS, schemaAnalysisReport, false, ""))
+	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport(queryissue.SECURITY_INVOKER_VIEWS_NAME, SECURITY_INVOKER_VIEWS_ISSUE, "", schemaAnalysisReport, false, ""))
```

Comment:
Let's filter by the type, not the type name? It's not great to be filtering on text that is used for display purposes, we should be checking an enum value.

(See the above call for regex functions as an example)

Reply:
i agree but if i use that facing issue that the issue is not getting reported.
Also i see there are other places also in this function which needs this change, lets do them together.

Reply:
done